.semaphore/semaphore.yml

```diff
@@ -11,52 +21,116 @@ global_job_config:
+          execution_time_limit:
+            minutes: 45
           commands:
-            - export BRANCH_NAME=$SEMAPHORE_GIT_BRANCH
-            - make ci
-            - >-
-              if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then 
-                make cd CONFIRM=true;
-              fi
-            - >-
-              if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then
-                make maybe-build-release;
-              fi
-      secrets:
-        - name: quay-robot-semaphore_v2
-        - name: operator-redhat-connect
+            - make build
+            - make dirty-check
+            # Store in cache for other builds in this run
+            - 'cache store bin-amd64-${SEMAPHORE_GIT_SHA} build/_output/bin'
+            - 'cache store go-pkg-cache-amd64-${SEMAPHORE_GIT_SHA} .go-pkg-cache'
+            - 'cache store go-mod-cache-amd64-${SEMAPHORE_GIT_SHA} ${HOME}/go/pkg/mod/cache'
+  - name: 'Build / Cache arm64 and ppc64le'
+    dependencies: [ "Build amd64" ]
+    task:
       prologue:
         commands:
-          - checkout
-          - >-
-            if [[ -v SEMAPHORE_GIT_PR_NUMBER ]]; then
-              unset QUAY_USERNAME QUAY_TOKEN; docker logout; docker logout quay.io;
-            else
-              docker login -u="$QUAY_USERNAME" -p="$QUAY_TOKEN" quay.io;
-            fi
-          - cache restore go-pkg-cache
-          - cache restore go-mod-cache
-      epilogue:
-        on_pass:
+          - cache restore go-pkg-cache-amd64-${SEMAPHORE_GIT_SHA}
+          - cache restore go-mod-cache-amd64-${SEMAPHORE_GIT_SHA}
+      jobs:
+        - name: Build arm64
+          execution_time_limit:
+            minutes: 45
+          commands:
+            - make register build ARCH=arm64
+            # Store in cache for other builds in this run
+            - 'cache store bin-arm64-${SEMAPHORE_GIT_SHA} build/_output/bin'
+            - 'cache store go-pkg-cache-arm64-${SEMAPHORE_GIT_SHA} .go-pkg-cache'
+            - 'cache store go-mod-cache-arm64-${SEMAPHORE_GIT_SHA} ${HOME}/go/pkg/mod/cache'
+        - name: Build ppc64le
+          execution_time_limit:
+            minutes: 45
+          commands:
+            - make register build ARCH=ppc64le
+            # Store in cache for other builds in this run
+            - 'cache store bin-ppc64le-${SEMAPHORE_GIT_SHA} build/_output/bin'
+            - 'cache store go-pkg-cache-ppc64le-${SEMAPHORE_GIT_SHA} .go-pkg-cache'
+            - 'cache store go-mod-cache-ppc64le-${SEMAPHORE_GIT_SHA} ${HOME}/go/pkg/mod/cache'
+  - name: 'Static / File Generation Checks'
+    dependencies: [ "Build amd64" ]
+    task:
+      jobs:
+        - name: Static / File Generation Checks, Vet
+          execution_time_limit:
+            minutes: 15
           commands:
-            - cache store go-pkg-cache .go-pkg-cache
-            - 'cache store go-mod-cache ${HOME}/go/pkg/mod/cache'
+            - make format-check validate-gen-versions fmt
+            - make dirty-check
+            - make vet
```

Comment:
I think most of these were accomplished from the `ci` target before, but I don't see a call to `test-crds`, I think that should be added here too.

Reply:
Right, I'll add that, thanks